### DISCUSSION TITLE
problemer em at instalere og config

### DISCUSSION THREAD

**Discussion by thebonden** [scaarup/aula#125] [Q&A]
hej jeg har fået intragrationen ind
men jeg kan simpel hen ikke lave et logind på aulasiden der kan jeg kun bruge mit id
det kunde være dejligt med en guide til opsærning fx video eller nogle foto
jeg har en søn der går i 0 klasse ville gærne lave en skærm/kontrolpanel i spisestugen hvor han kan se hvad de skal have på dagen 
nogle der har arfaringer med hvilken data der skal vises fx skema som liste eller dags skema eller ugeskema

jeg har kun haf HA kørne i ca 14 dage og er stadivæk i test fasen

**Comment by scaarup** [comment]
> hej jeg har fået intragrationen ind men jeg kan simpel hen ikke lave et logind på aulasiden der kan jeg kun bruge mit id det kunde være dejligt med en guide til opsærning fx video eller nogle foto jeg har en søn der går i 0 klasse ville gærne lave en skærm/kontrolpanel i spisestugen hvor han kan se hvad de skal have på dagen nogle der har arfaringer med hvilken data der skal vises fx skema som liste eller dags skema eller ugeskema
> 
> jeg har kun haf HA kørne i ca 14 dage og er stadivæk i test fasen

Hej.
Du skal logge ind i Aula med Unilogin for at bruge denne integration. Så det skal du have styr på først.
Skemaet hentes fra Aula in i HA og er en kalender-entitet. Dvs. du kan selv bestemme om du vil vise indholdet som en liste for 14 dage eller per uge - det er helt op til dig selv.

**Reply by thebonden** [reply]
tak så lykes det med uni login
hvis man nu vil lave det som et skema og ikke en liste som gæller for en uge af gangen
hvor dan gør man det
og  hvis man og så vil lave en hvor den kun viser for i dag

**Reply by scaarup** [reply]
> tak så lykes det med uni login
> 
> hvis man nu vil lave det som et skema og ikke en liste som gæller for en uge af gangen
> 
> hvor dan gør man det
> 
> og  hvis man og så vil lave en hvor den kun viser for i dag 

Lave hvad til et skema? Måske lige smid et par screenshots 🙂

**Reply by thebonden** [reply]
![image](https://github.com/scaarup/aula/assets/51397557/e7be892a-d090-47f2-b343-0f7e30a0b73b)

**Reply by scaarup** [reply]
> ![image](https://github.com/scaarup/aula/assets/51397557/e7be892a-d090-47f2-b343-0f7e30a0b73b)
> 
> 

Et Excel ark? Jeg forstår ikke hvad du vil.

**Reply by scaarup** [reply]
Du kan ikke bare bruge skemaet som det er?

![image](https://github.com/scaarup/aula/assets/8055470/2bc28763-ad55-4231-9d6d-a2305eea8e72)
![image](https://github.com/scaarup/aula/assets/8055470/26ccc9b4-cb3a-418e-bdd5-6baacd2325cb)![image](https://github.com/scaarup/aula/assets/8055470/e490f321-7425-4270-bb58-b05118f9fe2a)

![image](https://github.com/scaarup/aula/assets/8055470/a8793cfd-d3ae-47c1-ba82-be35e798ac81)
